src/components/feed/NodeDetails/NodeDetails.tsx

```diff
@@ -352,7 +352,7 @@ function getCommand(
     }
   }
 
-  let command = `$> docker run --rm -v $(pwd)/in:/incoming -v $(pwd)/out:/outgoing \\\n${dock_image} ${selfexec} `;
+  let command = `$> apptainer oci delete ${dock_image} & apptainer exec --bind $(pwd)/in:/incoming,$(pwd)/out:/outgoing ${dock_image} ${selfexec}`;
```

Comment:
- `apptainer oci delete` is not a necessary step.
- `&` is invalid syntax here. If you mean to run commands one after another, the syntax is `&&`

Reply:
Why is `apptainer oci delete` not necessary? We need to delete the container after execution as `--rm` does.